arch/arm64/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+set(
+  ARCH_ARM64_SOURCES
+  # Runtime
+  entry.S
+  kernel.c
+)
+
+add_kernel_module(
+  CORE # core module
+  NAME "arch_arm64"
+  SOURCES ${ARCH_ARM64_SOURCES}
+  INCLUDE_DIRS ${__ARCH_KERN_INCLUDES}
```

Comment:
uhh this shouldn't have a `__`

this is wrong in the `riscv` dir too

Reply:
I'll patch both, build system stuff I admittedly kind of just trust you to know what's going on